services/director-v2/src/simcore_service_director_v2/models/schemas/constants.py

```diff
@@ -1,11 +1,15 @@
+from typing import Final
+
 # dynamic services
 
-DYNAMIC_SIDECAR_SERVICE_PREFIX = "dy-sidecar"
-DYNAMIC_PROXY_SERVICE_PREFIX = "dy-proxy"
+DYNAMIC_SIDECAR_SERVICE_PREFIX: Final[str] = "dy-sidecar"
+DYNAMIC_PROXY_SERVICE_PREFIX: Final[str] = "dy-proxy"
+DYNAMIC_VOLUME_REMOVER_PREFIX: Final[str] = "dy-volrm"
+DY_SIDECAR_NAMED_VOLUME_PREFIX: Final[str] = "dyv"
 
 # label storing scheduler_data to allow service
 # monitoring recovery after director-v2 reboots
```

Comment:
for a later moment: since now you also have state in the dynamic sidecar, I guess you could also have an API for this.
Instead of having one part in the dy-sidecar, one part on the labels. just a thought.

Reply:
I'm not 100% sure to what you are referring here. What would the API return?

Reply:
what I mean is that now some part of the state is saved as a docker label, another part is saved inside the sidecar... etc etc.